src/tests/integration/test-data/foundry/test/ConstructorTest.t.sol

```diff
@@ -3,8 +3,34 @@ pragma solidity =0.8.13;
 
 import "forge-std/Test.sol";
 
+contract ImportedContract {
+    uint256 public count;
+
+    constructor() payable {
```

Comment:
I think that `payable` here will lead to the call value being symbolic?

Reply:
After some more consideration, I think that:
- if a function (constructors included) is `payable`, then the call value should be symbolic (and properly constrained) when starting CSE execution of that function;
- otherwise, the call value should be set to `0`.

If a non-payable function is called with a non-zero call value, the summary won't apply and the function will revert very quickly. I don't think we need a summary of that particular reverting behaviour.

From what I understand, in this test, the constructor should work with a symbolic call value, and the other functions should work with `0`. Could we please add:
- the `payable` attribute to one of the contract functions;
- the constructor itself to `test_kontrol_cse`?

Reply:
@PetarMax  How you described is already how it works, as far as I can tell. I've added payable to `add` and the constructor for `ImportedContract` and added the constructors to the `test_kontrol_cse`.